apps/studio/components/interfaces/Support/Success.tsx

```diff
@@ -29,8 +38,15 @@ const Success = ({ sentCategory = '' }: SuccessProps) => {
       <div className="flex items-center flex-col space-y-2">
         <h3 className="text-xl">Support request successfully sent!</h3>
         <p className="text-sm text-foreground-light">
-          We will reach out to you at <span className="text-foreground">{respondToEmail}</span>
+          We will reach out to you at <span className="text-foreground">{respondToEmail}</span>.
         </p>
+        {selectedProject !== 'no-project' && (
```

Comment:
Might be better to check for `project` being set here instead of selectedProject being some string